readme.md

```diff
@@ -1,2 +1,3 @@
 README
 
+submitting a PR
```

Comment:
looks good, maybe add a period

Reply:
@botany-lead sounds good - thank you the catch!